src/bp/core/routers/bots/nlu.ts

```diff
@@ -0,0 +1,248 @@
+import { Logger } from 'botpress/sdk'
+import * as sdk from 'botpress/sdk'
+import { EntityDefCreateSchema, IntentDefCreateSchema } from 'common/validation'
+import AuthService, { TOKEN_AUDIENCE } from 'core/services/auth/auth-service'
+import { EntityService } from 'core/services/nlu/entities-service'
+import { IntentService } from 'core/services/nlu/intent-service'
+import { WorkspaceService } from 'core/services/workspace-service'
+import { RequestHandler, Router } from 'express'
+import { validate } from 'joi'
+import _ from 'lodash'
+import yn from 'yn'
+
+import { CustomRouter } from '../customRouter'
+import { checkTokenHeader, needPermissions } from '../util'
+
+const removeSlotsFromUtterances = (utterances: { [key: string]: any }, slotNames: string[]) =>
+  _.fromPairs(
+    Object.entries(utterances).map(([key, val]) => {
+      const regex = new RegExp(`\\[([^\\[\\]\\(\\)]+?)\\]\\((${slotNames.join('|')})\\)`, 'gi')
+      return [key, val.map(u => u.replace(regex, '$1'))]
+    })
+  )
+
+export class NLURouter extends CustomRouter {
+  private _checkTokenHeader: RequestHandler
+  private _needPermissions: (operation: string, resource: string) => RequestHandler
+
+  constructor(
+    private logger: Logger,
+    private authService: AuthService,
+    private workspaceService: WorkspaceService,
+    private entityService: EntityService,
+    private intentService: IntentService
```

Comment:
Can you provide the NLUService here, instead? Also, we should remove the name "service", since we can no longer inject them elsewhere. Then we can simply call `nluService.intents` or `nluService.entities`.